app/Policies/AchievementAuthorPolicy.php

```diff
@@ -65,4 +69,15 @@ public function forceDelete(User $user, AchievementAuthor $achievementAuthor): b
     {
         return false;
     }
+
+    public function canUpsertTask(User $user, AchievementAuthorTask $task): bool
+    {
+        // Artists can only give artwork credit.
+        if ($user->roles->count() === 1 && $user->hasRole(Role::ARTIST)) {
+            return $task === AchievementAuthorTask::Artwork;
+        }
```

Comment:
I don't think you want to limit this to users with a single role. What if someone is an Artist and a Play Tester?

Reply:
Yep, this is a logic error. Resolved in latest.